src/os.c

```diff
@@ -237,13 +242,27 @@ void _mi_os_init() {
 
 #else  // generic unix
 
+static void os_detect_transparent_huge_pages(void) {
+#if defined(MADV_HUGEPAGE)
+  int fd = open("/sys/kernel/mm/transparent_hugepage/enabled", O_RDONLY);
```

Comment:
Is this path always fixed? I read [here](https://docs.mongodb.com/manual/tutorial/transparent-huge-pages/) that it can also be `/sys/kernel/mm/redhat_transparent_hugepage/enabled` for example?

Reply:
ah good to know